### PR TITLE
Add a basic package page.

### PR DESCRIPTION
**Pull request by rictic**
Fixes https://github.com/webcomponents/webcomponents.org/issues/1444

### CODE REVIEW

File: packages/site-client/src/pages/package/wco-package-page.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  version: string;
+  elements: CustomElement[];
+}
+
+@customElement('wco-package-page')
+export class WCOPackagePage extends WCOPage {
+  static styles = [
+    WCOPage.styles,
+    css`
+      .full-screen-error {
```
Comment (justinfagnani) [inline packages/site-client/src/pages/package/wco-package-page.ts:27]: We should eventually move this to common styles

File: packages/site-server/src/lib/catalog/routes/package/package-route.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }`,
+  });
+
+  if (result.errors !== undefined && result.errors.length > 0) {
+    throw new Error(result.errors.map((e) => e.message).join('\n'));
+  }
+  const {data} = result;
+  const packageVersion = data.package?.version;
+  if (packageVersion === undefined) {
+    throw new Error(`No such package version: ${packageName}`);
```
Comment (justinfagnani) [inline packages/site-server/src/lib/catalog/routes/package/package-route.ts:64]: I think we should probably be rendering a 404 in these cases. TODO?

**Review by justinfagnani — APPROVED** [review]

File: packages/site-client/src/pages/package/wco-package-page.ts (hunk 1 above)
Comment (rictic) [reply packages/site-client/src/pages/package/wco-package-page.ts:27]: +1, done

File: packages/site-server/src/lib/catalog/routes/package/package-route.ts (hunk 2 above)
Comment (rictic) [reply packages/site-server/src/lib/catalog/routes/package/package-route.ts:64]: +1, TODO added

**Pull request merged**